pkgs/applications/version-management/radicle-cli/default.nix

```diff
@@ -2,86 +2,38 @@
 , stdenv
 , fetchFromGitHub
 , rustPlatform
-, pkg-config
-, cmake
-, installShellFiles
-, asciidoctor
 , DarwinTools
-, openssl
-, libusb1
 , AppKit
-, git
-, openssh
-, testers
-, radicle-cli
 }:
 
 rustPlatform.buildRustPackage rec {
   pname = "radicle-cli";
-  version = "0.6.1";
+  version = "0.8.0";
 
   src = fetchFromGitHub {
     owner = "radicle-dev";
-    repo = pname;
-    rev = "v${version}";
-    sha256 = "sha256-LS6zYpMg0LanRL2M8ioGG8Ys07TPT/3hP7geEGehwxg=";
+    repo = "heartwood";
+    rev = "b27313bc5c703c8a25343176c620c8652b815abc";
```

Comment:
Hey @amesgen, we currently aren't updating the `version` since we didn't do a proper release of `0.8.0` yet.
Once heartwood is stable we will again follow semver conventions.

I think marking it with `-unstable-` and the date of the fetched commit is a good solution. 👍